Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -456,6 +456,8 @@ static void setVideoDecoderBehaviors(OptionSet<VideoDecoderBehavior> videoDecode
 #endif
 
 #if USE(APPKIT)
+    [NSApplication sharedApplication];
```

Comment:
I think this could use an inline comment since it seems a bit strange as-is unless you know the backstory, but that's just imo